packages/react-components/react-swatch-picker-preview/src/components/ColorSwatch/useColorSwatchStyles.styles.ts

```diff
@@ -0,0 +1,102 @@
+import { makeResetStyles, makeStyles, mergeClasses, shorthands } from '@griffel/react';
+import type { SlotClassNames } from '@fluentui/react-utilities';
+import type { ColorSwatchSlots, ColorSwatchState } from './ColorSwatch.types';
+import { tokens } from '@fluentui/react-theme';
+
+export const colorSwatchClassNames: SlotClassNames<ColorSwatchSlots> = {
+  root: 'fui-ColorSwatch',
+};
+
+export const swatchCSSVars = {
+  swatchColor: `--fui-SwatchPicker--color`,
+};
+
+const { swatchColor } = swatchCSSVars;
+
+/**
+ * Styles for the root slot
+ */
+const useStyles = makeResetStyles({
+  boxSizing: 'border-box',
+  ...shorthands.border('none'),
+  ...shorthands.padding(0),
```

Comment:
```suggestion
  border: 'none',
  padding: 0,
```

Reply:
can we add griffel lint rules for these scenarios ?

Reply:
@Hotell yes, can you please create an issue? 🙏